packages/calling-component-bindings/review/calling-component-bindings.api.md

```diff
@@ -123,6 +123,8 @@ activeErrors: ActiveError[];
 activeErrors: ActiveError[];
 }>;
 
+// Warning: (ae-incompatible-release-tags) The symbol "GetSelector" is marked as @public, but its signature references "videoGallerySelector" which is marked as @beta
```

Comment:
This should be fixed.. but not sure what should happen here, only part of the GetCallingSelector is returning a beta item....